### PR TITLE
tests: only output last 200 lines of various logs

### PR DESCRIPTION
**Pull request by ianvernon**
This will help us avoid having disk fill up on Jenkins.

Signed-off by: Ian Vernon <ian@cilium.io>

### CODE REVIEW

**Comment by joestringer** [comment]
Are these otherwise accessible after the fact through the tarball?

**Comment by ianvernon** [comment]
@joestringer yes all logs are packed in the tarball

**Review by joestringer — APPROVED** [review]
LGTM

File: tests/k8s/helpers.bash [hunk 1]
```diff
@@ -16,14 +16,14 @@ function abort {
 
 	cilium_id=$(docker ps -aql --filter=name=cilium-agent)
 	echo "------------------------------------------------------------------------"
-	echo "                            Cilium logs"
-	docker logs ${cilium_id}
+	echo "                            Cilium logs (last 200 lines)"
+	docker logs ${cilium_id} | tail -n 200
```
Comment (aanm) [inline tests/k8s/helpers.bash]: `docker logs --tail 200`?

File: tests/k8s/helpers.bash [hunk 2]
```diff
@@ -16,14 +16,14 @@ function abort {
...
 	echo ""
 	echo "------------------------------------------------------------------------"
 
     echo "------------------------------------------------------------------------"
-    echo "                            L7 Proxy logs"
-    cat /var/lib/cilium/proxy.log
+    echo "                            L7 Proxy logs (last 200 lines)"
+    cat /var/lib/cilium/proxy.log | tail -n 200
```
Comment (aanm) [inline tests/k8s/helpers.bash]: nit: `tail -n 200 /var/lib/cilium/proxy.log`

**Review by aanm — DISMISSED** [review]

File: tests/k8s/helpers.bash (hunk 1 above)
Comment (ianvernon) [reply tests/k8s/helpers.bash]: Didn't know this option existed, fixed.

File: tests/k8s/helpers.bash (hunk 2 above)
Comment (ianvernon) [reply tests/k8s/helpers.bash]: Fixed.

**Review by aanm — APPROVED** [review]

**Pull request merged**